### PR TITLE
first attempt for amazon science

### PR DESCRIPTION
**Pull request by dafnapension**
Card preparer  for   AmazonScience/massive
a variation on theme  ethos.py

### CODE REVIEW

File: prepare/cards/amazon_mass.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                InputOutputTemplate(
+                    input_format="Given this sentence: {utt}. Classify it into one of the following classes. Choices: {choices}.",
+                    output_format="{intent}",
+                    postprocessors=["processors.take_first_non_empty_line"],
+                )
+            ]
+        ),
+    )
+    if lang == "en-US":
+        test_card(card, debug=True)
```
Comment (yoavkatz) [inline prepare/cards/amazon_mass.py:51]: Hi.  By default, debug should be false, so speed up regressions.

File: prepare/cards/amazon_mass.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+from datasets import get_dataset_config_names
+
+langs = get_dataset_config_names("AmazonScience/massive")
+# now langs is the list of all languages showing in the dataset
+
+from datasets import load_dataset_builder
+
+ds_builder = load_dataset_builder("AmazonScience/massive")
+classlabels = ds_builder.info.features["intent"]
```
Comment (yoavkatz) [inline prepare/cards/amazon_mass.py:9]: Hi. This is something that lends itself to a development worth adding to unitxt.

Create an Operator that can be used to extract all values of a specific field.

ExtractMostCommonFieldValues(from_stream="train", from_field="intent",to_field = "choices")

**Review by yoavkatz — CHANGES_REQUESTED** [review]
Please see feedback below.

File: prepare/cards/amazon_mass.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        loader=LoadHF(path="AmazonScience/massive", name=lang),
+        preprocess_steps=[
+            MapInstanceValues(mappers=mappers),
+            AddFields(
+                fields={
+                    "choices": classlabels.names,
+                }
+            ),
+        ],
+        task=FormTask(inputs=["choices", "utt"], outputs=["intent"], metrics=["metrics.accuracy"]),
```
Comment (yoavkatz) [inline prepare/cards/amazon_mass.py:39]: utt -> text
 choice -> classes
 "sentence" -> text_type
 "intent" -> type_of_class

File: prepare/cards/amazon_mass.py (hunk 2 above)
Comment (dafnapension) [reply prepare/cards/amazon_mass.py:9]: discussed and put into issue #297

File: prepare/cards/amazon_mass.py (hunk 3 above)
Comment (dafnapension) [reply prepare/cards/amazon_mass.py:39]: discussed and put into issue https://github.com/IBM/unitxt/issues/297

File: prepare/cards/amazon_mass.py (hunk 1 above)
Comment (dafnapension) [reply prepare/cards/amazon_mass.py:51]: changed to debug=False

**Comment by dafnapension** [comment]
Created a new pull request, without the yaml file and all the card files,
Please approve the new one: #300

**Pull request closed without merging**